source

```diff
@@ -102671,10 +102678,12 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>data state</span>. Emit the current tag token.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd><span data-x="parse-error-eof-in-attribute-value">eof-in-attribute-value</span><span>parse
+   error</span>. Emit an end-of-file token.</dd>
 
    <dt>Anything else</dt>
-   <dd><span>Parse error</span>. <span>Reconsume</span> in the <span>before attribute name
+   <dd>This is a <dfn data-x="parse-error-missing-whitespace-after-attribute-value">missing-whitespace-after-attribute-value</dfn>
```

Comment:
Let me look at the tests, because as I was going over then, some didn't match very well the description. If they make more sense with this suggestion, I will update, if not  I would tell you then doesn't seem to fit.